lib/json-schema/attributes/formats/ip.rb

```diff
@@ -10,6 +10,7 @@ def self.validate(current_schema, data, fragments, processor, validator, options
 
         begin
           ip = IPAddr.new(data)
+        rescue IPAddr::InvalidAddressError
```

Comment:
I considered adding a comment here, since it is confusing, but left explanation in the commit message instead. Happy to add an explicit comment if preferable.